assets/js/modules/adsense/components/common/ErrorNotice.js

```diff
@@ -31,9 +31,9 @@ export default function ErrorNotice() {
 
 	// Do not display if no error, or if the error is for missing scopes, or if
 	// it yields an account status, in which case it is an "expected" error.
-	if ( ! error || error.code === PERMISSION_SCOPE_ERROR_CODE || undefined !== errorToStatus( error ) ) {
+	if ( ! error || isPermissionScopeError( error ) || undefined !== errorToStatus( error ) ) {
 		return null;
 	}
 
-	return <ErrorText message={ error.message } />;
+	return <ErrorText message={ error.message } reconnectURL={ error.data.reconnectURL } />;
```

Comment:
Probably a bit safer to use `error.data?.reconnectURL` here, in case `error.data` is not present.